source/settings/core.rst

```diff
@@ -3463,6 +3463,14 @@ See :ref:`settings` for list of all setting groups.
    The path to the stats-writer socket.
 
 
+.. dovecot_core:setting:: submission_add_received_header
+   :added: v2.3.19,v3.0.0
+   :default: yes
+   :values: @boolean
+
+   Controls if "Received:" header should be added to submitted mails.
```

Comment:
Is this added only by the submission backend server, or also by each proxy?

Reply:
the change itself is only in src/submission/submission-{command,settings}.*
unfortunately I don't know yet how proxies work, beyond the general and generic concept of proxy,
so I am not able to reply, Timo
(rather, I'd like to learn it)